server/views/templates/explore-landing.njk

```diff
@@ -1,6 +1,8 @@
 {% extends "pages/explore-landing.njk" %}
 
 {% block body %}
+  {% component 'page-description', { title: 'Explore landing', modifiers: ['hidden'] } %}
```

Comment:
What SEO does explore landing offer?
Shouldn't we be describing the page to someone who comes here for the first time with a screenreader?